docs/explanation/tendermint2.md

```diff
@@ -0,0 +1,121 @@
+---
+id: tendermint2
+---
+
+# Tendermint2
+
+**Disclaimer: Tendermint2 is currently part of the Gno monorepo for streamlined development.**
+
+**Once Gno.land is on the mainnet, Tendermint2 will operate independently, including for governance, on https://github.com/tendermint/tendermint2.**
+
+## Mission
+
+* Make awesome software with modular components.
+* Crypto P2P Swiss armyknife for human liberation.
```

Comment:
I disagree. I don't see the point in encouraging doing everything. Instead, I believe in maintaining our opinionated and minimalist philosophy, which allows us to do anything but not everything.

Reply:
I believe Onbloc took this from the official TM2 readme:
https://github.com/tendermint/tendermint2#mission

I've dropped this section for the time being (this PR), so we can define it carefully:
[e8f1460](https://github.com/gnolang/gno/pull/1046/commits/e8f14607737198046a4171e83a69b4cea902b9c0)